.github/workflows/regression.yml

```diff
@@ -0,0 +1,44 @@
+name: Run regression tests
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/regression.yml"
+      - "package*.json"
+      - "test/**"
+      - "examples/**"
+      - "!examples/landmarks/**"
+  push:
+    paths:
+      - ".github/workflows/regression.yml"
+      - "package*.json"
+      - "test/**"
+      - "examples/**"
+      - "!examples/landmarks/**"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Run Ava on changed files in PR
+        run: ./scripts/regression-tests.sh
+        if: github.event_name == 'pull_request'
+        env:
+          COMMIT_RANGE: "origin/${{ github.base_ref }}...${{ github.sha }}"
```

Comment:
We need to add a env variable here to extend the wait time for the CI -- `TEST_WAIT_TIME=1000` was the wait time we set in travis.

Reply:
Odd, looks like the default is supposed to be higher https://github.com/avajs/ava/blob/master/docs/07-test-timeouts.md#test-timeouts but I'll take a look

Reply:
Oh you know I missed this, this is a different time out than the default ava time out, it's a timeout used for a different purpose in the tests. The timeout is related to how long we might want to wait for the DOM to change. I'm going to push that variable and see if it works!